ministryofjustice/correspondence_tool_staff
# Ct 3922 create sar ir case

## Description
Adds page to enter details of the SAR IR with relevant details pulled from the original SAR.

Note I am going to rails another PR with page objects for the feature tests. This PR was getting big so I wanted to get it reviewed before I made further changes.

## Self-review checklist



### Screenshots
Adds new page where user can enter the SAR IR case details:

<img width="333" alt="Screenshot 2021-11-30 at 15 58 52" src="https://user-images.githubusercontent.com/13377553/144082146-0cd401ec-3431-4f19-af25-287d5a04dbd3.png">

### Related JIRA tickets
[CT-3922](https://dsdmoj.atlassian.net/browse/CT-3922)
[CT-3923](https://dsdmoj.atlassian.net/browse/CT-3923)

### Manual testing instructions
- copy and existing SAR case number
- click create case
- enter copied SAR number
- Continue
- Add the extra details to the page
- Continue - should be directed to the case show page (not the case details partial for this is a placeholder at the moment)
